cmd/mtls.go

```diff
@@ -101,5 +101,6 @@ func init() {
 	MTLSCmd.MarkFlagRequired("kubernetes")
 	MTLSCmd.AddCommand(ExportCMD)
 	MTLSCmd.AddCommand(ExpiryCMD)
+	MTLSCmd.AddCommand(RenewCertificateCmd)
```

Comment:
Should the `mtls` command reside in `mtls.go`, just like `export` and `expiry` commands?

Reply:
I created new file on purpose. IMO it would have made mtls file bulky and hard to read. At the same time line 104 give us idea about the new subcommand.

Reply:
I would vote for creating a new directory for mtls then. 

Example: https://github.com/argoproj/argo-workflows/blob/master/cmd/argo/commands/cron/root.go